src/js/outreach/apps/opt-in_app.js

```diff
@@ -0,0 +1,68 @@
+import Backbone from 'backbone';
+import App from 'js/base/app';
+
+import { optInPostRequest } from 'js/outreach/entities';
+
+import {
+  OptInView,
+  SuccessView,
+  ErrorView,
+} from 'js/outreach/views/opt-in_views';
+
+import {
+  DialogView,
+} from 'js/outreach/views/dialog_views';
+
+const StateModel = Backbone.Model.extend({
+  defaults: {
+    firstName: '',
+    lastName: '',
+    dob: '',
+    phoneNumber: '',
+    email: '',
+  },
+});
+
+export default App.extend({
+  StateModel,
+  onStart() {
+    const dialogView = new DialogView();
+    this.dialogViewRegion = dialogView.getRegion('content');
+
+    this.showOptInView();
+
+    this.showView(dialogView);
+  },
+  showOptInView() {
+    const optInView = new OptInView({ model: this.getState() });
+
+    this.listenTo(optInView, 'click:submit', () => {
+      // validate input fields and pass down any errors to the view before making the api request?
+
+      optInPostRequest({
+        inputData: this.getState,
+      })
+        .then(() => {
+          this.showSuccessView();
+        })
+        .catch(() => {
+          this.showErrorView();
+        });
+    });
+
+    this.dialogViewRegion.show(optInView);
```

Comment:
since `dialogView` was used in `showView` this is the same as `this.showChildView('content', optInView)`

Reply:
Good call, I updated it to that.